src/celeritas/optical/model/RayleighModel.cc

```diff
@@ -111,6 +111,19 @@ void RayleighModel::build_mfps(OptMatId mat, MfpBuilder& build) const
         }
         build(grid);
     }
+    else
+    {
+        auto mat_view = input_.materials->get(mat);
+        auto rindex_calc = mat_view.make_refractive_index_calculator();
+        auto energies = rindex_calc.grid().values();
+        if (!energies.empty())
+        {
+            inp::Grid g;
+            g.x.assign(energies.begin(), energies.end());
```

Comment:
Better to just make two points:
```suggestion
            g.x = {energies.front(), energies.back()};
```

Reply:
Good point - i was thinking for the case when the shape of the grid had to be same as the other optical tables, but i am fine reducing it to two points if that is not the concern.